src/Components/ItemList.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import Item from './Item'
+
+const ItemList=({props}) =>{
+    return (
+        <div>
+            <h2>My Garage Sale</h2>
+            {props.data.map(item => {
+                return (
+                    <div className="Items" >
+                        <Item item={item} />
+                        <button onClick={() => props.addToCart(item)}>Add To Cart</button>
```

Comment:
There's already a button created in the Item component, no need to create buttons in both locations! pass the addToCart function as a prop to the Item component so that the Item component can use addToCart as a click handler for the button.